src/Sound/Tidal/Pattern.hs

```diff
@@ -47,8 +47,8 @@ type Time = Rational
 newtype Note = Note Double deriving (Typeable, Data, Generic, Eq, Ord, Show, Enum, Num, Fractional)
```

Comment:
you could write `newtype Note = Note { unNote :: Double }`.

Reply:
done :)